### ISSUE TITLE
Revit_Toolkit: Not pulling roof panel

### ISSUE THREAD

**Issue by ebowyer** [BHoM/Revit_Toolkit#351]

**Comment by michaldengusiak** [comment]
yes correct this is already logged here and we starting whole process
https://github.com/BHoM/Revit_Toolkit/issues/312

**Comment by ebowyer** [comment]
It says on that that it should now support the roof too... but it doesn't?

**Comment by michaldengusiak** [comment]
see my last comment at the end...all boxes are not ticked and therefore need to be re-fix
